projects/igniteui-angular/src/lib/select/select-navigation.directive.ts

```diff
@@ -17,11 +17,11 @@ export class IgxSelectItemNavigationDirective extends IgxDropDownItemNavigationD
 
     /** Captures keydown events and calls the appropriate handlers on the target component */
     handleKeyDown(event: KeyboardEvent) {
-        if (!event || event.shiftKey) {
```

Comment:
Was the shift here excluded for a reason before?

Reply:
If the Shift check is present, it will not allow to navigate (when collapsed) through the items pressing Shift + Up/Down arrow keys. I prefer to revert this, as we have never specified it is possible to use Shift + Up/Down arrow keys for this interaction (Currently, the user can navigate and select list items with up and down arrow keys.).